### PR TITLE
Add coarse focus to evening calibrations

### PR DESCRIPTION
**Pull request by AnthonyHorton**
Adds a coarse focus off all cameras to the evening calibration state, after flat fielding and nautical twilight, in order to implement the first item in #69.

Also took the opportunity to refactor the waiting for a specific Sun altitude into a utility function, and edited a few message strings ('sunset' -> 'twilight').

### CODE REVIEW

**Review by wtgee — CHANGES_REQUESTED** [review]
Hmm, a lot of the same thing is going on over here: https://github.com/panoptes/POCS/pull/541/files#diff-a318b148222fe28a50df3aa2824429fdR522. It's a little more convoluted over there as it is pulled into the pocs machinery itself and reads from the config.

It keeps the `pocs` objects out of utils file, which I think we should enforce. The `pocs` items are only used in state files as it is.

Can we just add the coarse focus after the flat-fielding for this PR and then we'll add in the appropriate wait after?

**Comment by AnthonyHorton** [comment]
Ah, wasn't aware of the overlap with a WIP in POCS itself. Was just keen to make sure it was dark enough for the coarse focus to work, and felt I shouldn't just add a 3rd nearly identical `while` loop to `calibrating.py`. I would have just passed the `observatory` or `observer` to my utility function, but then I wouldn't be able to keep the `pocs.say`.

**Comment by AnthonyHorton** [comment]
@wtgee Compromise temporary solution, move the `wait_for_sun_alt` utility function into `huntsman-pocs/states/calibrating.py` instead of `huntsman-pocs/utils/flats.py`?   Keeps the `pocs` object out of `utils` and avoids any overlap with the POCS WIP while providing the desired functionality in the meantime.  Once the WIP gets merged into POCS we can chuck this out of the `huntsman-pocs` state machine and use the new functions.  Going to want to do a bunch of that sort of thing anyway to stop `huntsman-pocs` duplicating functionality that's been added to POCS.

**Comment by AnthonyHorton** [comment]
Doing some tinkering to get Travis and coverage working. Will spin out into a separate PR.

File: states/huntsman/calibrating.py [hunk 1]
```diff
@@ -3,6 +3,42 @@
...
+        min_altitude (astropy.units.Quantity, optional):
+        max_altitude (astropy.units.Quantity, optional):
+        message (str):
+    """
+    if min_altitude is None and max_altitude is None:
+        raise ValueError("At least one of min_altitude & max_altitude must be given")
+    if min_altitude is None:
+        min_altitude = -90
+    if max_altitude is None:
+        max_altitude is +90
```
Comment (wtgee) [inline states/huntsman/calibrating.py]: `max_altitude = 90`

**Review by wtgee — APPROVED** [review]
LGTM although I see tests are failing.

File: states/huntsman/calibrating.py (hunk 1 above)
Comment (AnthonyHorton) [reply states/huntsman/calibrating.py]: OK.

**Comment by AnthonyHorton** [comment]
Passing now, thanks to #79!

**Pull request merged**